library/src/scala/quoted/Quotes.scala

```diff
@@ -2524,6 +2533,11 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
         def typeSymbol: Symbol
         def termSymbol: Symbol
         def isSingleton: Boolean
+
+       /** This type seen as if it were the type of a member of prefix type `self` declared in class `member.owner`.
```

Comment:
I think this description is incorrect, we don't return a version of "This type" but of the type of the member.
```suggestion
       /** The type of `member` as seen from prefix `self`
        *
        *  @pre `member` is an accessible member of `self`
```